docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,341 @@
+---
+sidebar_label: MLflow 3.0
+sidebar_position: 1
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Platform from "@site/src/content/platform.mdx"
+
+# MLflow 3.0
+
+Discover the next generation of MLflow, designed to streamline your AI experimentation and accelerate your journey from idea to production. MLflow 3.0 brings cutting-edge support for GenAI workflows, enabling seamless integration of generative AI models into your projects.
+
+## What is MLflow 3.0?
+
+The best AI experimentation platform for building high quality models and GenAI agents! MLflow 3.0 introduces a variety of exciting improvements to MLflow Tracking and Tracing, including:
+- First class model support in MLflow tracking with **LoggedModel**
+- **Model-centric** information architecture for params and metrics in MLflow tracking
+- Enhanced **tracing** experience with lineage tracking for LoggedModels
+
+Model Tracking helps you create and evaluate different model/agent configurations in your experiments, LoggedModel helps you find the best ones for production usage, and tracing helps enhance observability in your model or GenAI applications.
+
```

Comment:
Can we add a pseudo table of contents so that users can drive to the sections "MLflow 3 for GenAI" "MLflow 3 for Deep Learning" as an indicator that "this isn't just for GenAI". Perhaps consolidate the GenAI examples together and have the DL example at the bottom.

Reply:
<img width="290" alt="image" src="https://github.com/user-attachments/assets/337fd637-496b-4755-90b7-eeeee5539732" />
The ToC on the right is quite straightforward 🤔 Do we need an extra list here? Maybe if we move them into separate pages later and adding a table of contents here makes more sense?